commands/push.go

```diff
@@ -95,3 +107,14 @@ func pushStack(services *stack.Services, queueDepth int) {
 	wg.Wait()
 
 }
+
+func validateImages(services *stack.Services) string {
+	errMsg := make([]string, 0)
+	for name, function := range services.Functions {
+
+		if !function.SkipBuild && !strings.Contains(function.Image, `/`) {
```

Comment:
Do you need to handle the case where the image only contains the `/`, for example `/imgfn`?

Reply:
I don't think so? In that instance Docker will still error.

Reply:
Thoughts?

Reply:
This is the error message that docker provides

```
invalid reference format
2018/02/22 09:21:09 ERROR - Could not execute command: [docker push /movies:0.1]
```

I think that's pretty clear, but do you think the `faas-cli` should provide something?

Reply:
I just don't like the `invalid reference format` that results from the docker stderr when we are in a position to absorb it.

Its by no means a blocker however, just a personal preference for actionable output.